Formula/ffmpeg@5.rb

```diff
@@ -0,0 +1,139 @@
+class FfmpegAT5 < Formula
```

Comment:
On second thoughts, I think `ffmpeg@5.1` may be a better name. It is not consistent with `ffmpeg@4`, but 5.0 and 5.1 are [separate branches](https://github.com/FFmpeg/FFmpeg/branches) and we can't tell if there will be a non-LTS 5.2 branch. (Not sure if we'll prefer that to 5.1 series.)